stable/shell/templates/shell-configmap.yaml

```diff
@@ -29,6 +29,7 @@ data:
 {{- if and .Values.v3io.tsdbctl .Values.v3io.tsdbctl.download }}
     curl -sSL -o ${IGUAZIO_HOME}/bin/tsdbctl {{ .Values.v3io.tsdbctl.download }}
     chmod +x ${IGUAZIO_HOME}/bin/tsdbctl
+    echo alias tsdb=\"tsdbctl -s {{ .Values.webapi.url }} -k {{ .Values.v3io.accessKey }}\"
```

Comment:
same here

Reply:
Please check the latest revision. It should address all the above.